lib/compat/wordpress-6.0/block-patterns.php

```diff
@@ -171,6 +171,13 @@ function gutenberg_register_theme_block_patterns() {
 						}
 					}
 
+					// Translate the pattern metadata.
+					$text_domain           = wp_get_theme()->get( 'TextDomain' );
+					$pattern_data['title'] = translate_with_gettext_context( $pattern_data['title'], 'Title of the pattern', $text_domain );
```

Comment:
For some reason, the strings are not being translated and they return the English version.

This is what I've checked:

- Is the string and context correct? They are.
- Is the textdomain for the theme loaded? It is, according to `is_textdomain_loaded( $text_domain )`.
- Is the `.mo` file is loaded correctly? It is according to the following tests:
  - Modifying a different string in the po file, then regenerate the mo file, and verify that I see the changes in the UI.
  - Inspecting that the translations for the domain actually contain the pattern strings by checking `get_translations_for_domain( $text_domain );`.

Reply:
@swissspidy would you have any hint of what could be happening?

Reply:
@oandregal Your MO file does not contain the context for the strings.

You can confirm this by running `translate( $pattern_data['title'], $text_domain )`.

So the MO file needs to be fixed.

Reply:
Thanks!